### DISCUSSION TITLE
Problem with seeding

### DISCUSSION THREAD

**Discussion by ikarossvedberg** [beakerbrowser/beaker#1834] [Q&A]
Hello,

I've been using beaker to make mock-ups of websites, wich I can send as previews to clients using _hashbase.io_
It's been working great up until recently, when I suddenly could not seed my own sites. I updated to the new Beaker browser but my issue remains. Also, I cannot seed other peoples sites either.

When I click the seed button I just see a spinning wheel. I've tried different networks, restarted Beaker several times etc.

What am I missing?

Below: What I see when I try to seed.

<img width="200" alt="Skärmavbild 2020-10-15 kl  17 33 59" src="https://user-images.githubusercontent.com/38424783/96152805-799fdf00-0efc-11eb-9849-3d057d943257.png">

**Comment by despens** [comment]
Currently, the last version where the peers widget works is Beaker.Browser-1.0.0-prerelease.4, I recommend using that one in place of the current prerelease.7.

**Reply by pfrazee** [reply]
Yeah that's accurate, we just broke the feature. Will be fixed in the upcoming 1.0 release.
